Package/gulpfile.js

```diff
@@ -543,35 +570,50 @@ const patchPackageVersion = async () => {
 
     const packageJsonPath = '../Modules/@babylonjs/react-native/package.json';
     const packageJsonPathWindows = '../Modules/@babylonjs/react-native-windows/package.json';
+    const packageJsonPathiOSAndroid = '../Modules/@babylonjs/react-native-iosandroid/package.json';
     const packageJson = JSON.parse(fs.readFileSync(packageJsonPath));
     const packageJsonWindows = JSON.parse(fs.readFileSync(packageJsonPathWindows));
+    const packageJsoniOSAndroid = JSON.parse(fs.readFileSync(packageJsonPathiOSAndroid));
 
     if (version == '0.64') {
-      packageJson.peerDependencies['react-native'] = '>=0.63.1 <0.65.0';
       packageJsonWindows.peerDependencies['react-native'] = '>=0.63.1 <0.65.0';
+      packageJsoniOSAndroid.peerDependencies['react-native'] = '>=0.63.1 <0.65.0';
       packageJsonWindows.peerDependencies['react-native-windows'] = '>=0.63.1 <0.65.0';
     } else {
-      packageJson.peerDependencies['react-native'] = '>=0.65.0';
       packageJsonWindows.peerDependencies['react-native'] = '>=0.65.0';
+      packageJsoniOSAndroid.peerDependencies['react-native'] = '>=0.65.0';
       packageJsonWindows.peerDependencies['react-native-windows'] = '>=0.65.0';
     }
 
+    // release version
+    const releaseVersion = (process.argv[4] == '--releaseVersion') ? process.argv[5] : ((process.argv[5] == '--releaseVersion') ? process.argv[6] : '');
```

Comment:
Does this mean when testing the package build locally you now have to pass in a version as an arg?

Reply:
I changed the function a bit so you can specify a react-native version and/or the package release version. Both are optional now.